docs/en/operation/User-CLI.md

```diff
@@ -813,7 +813,7 @@ $ ./bin/alluxio fs cp /hdfs/file1 /s3/
 ### distributedCp
 
 The `distributedCp` command copies a file or directory in the Alluxio file system distributed across workers
-using the job service.
+using the job service. User will get a job control ID after the command submission is successful.
```

Comment:
User will only get a job control ID if it is async  mode right?

Reply:
Yes